src/css/modules/tag.css

```
@@ -0,0 +1,113 @@
+  cursor: pointer;
+  opacity: 0.7;
+  transition:
+    opacity var(--lay-anim-duration-fast),
+    background-color var(--lay-anim-duration-fast);
+}
+.lay-tag-close:hover {
+  opacity: 1;
+  background-color: rgba(0, 0, 0, 0.15);
+}
+.lay-tag-close:focus-visible {
+  outline: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)